definitions/json/CaseField.json

```diff
@@ -1396,7 +1396,7 @@
     "CaseTypeID": "ET_EnglandWales",
     "ID": "icDateCompleted",
     "Label": "Date completed:",
-    "FieldType": "Text",
+    "FieldType": "Date",
```

Comment:
it might complain about this because it's changing the data type

Reply:
As discussed offline, config can be imported fine after changing text to date